sdk/clientcore/core/src/main/java/io/clientcore/core/serialization/xml/implementation/aalto/stax/InputFactoryImpl.java

```diff
@@ -19,24 +19,16 @@
 import io.clientcore.core.serialization.xml.implementation.aalto.in.CharSourceBootstrapper;
 import io.clientcore.core.serialization.xml.implementation.aalto.in.ReaderConfig;
 
-import javax.xml.stream.EventFilter;
-import javax.xml.stream.StreamFilter;
-import javax.xml.stream.XMLEventReader;
-import javax.xml.stream.XMLInputFactory;
-import javax.xml.stream.XMLReporter;
-import javax.xml.stream.XMLResolver;
 import javax.xml.stream.XMLStreamException;
```

Comment:
aren't these imports going to be problematic since the whole package is not available on android?